code/game/objects/effects/spawners/ghostroles.dm

```diff
@@ -190,17 +190,16 @@
 
 /obj/effect/mob_spawn/human/lonestar_ert
 	name = "deployment sleeper"
-	desc = "A sleeper, with an unconscious body inside. The occupant seems to be an Auditor."
+	desc = "A sleeper, with an unconscious body inside. The occupant seems to be a Lonestar IA Agent."
 	mob_name = "a human"
 	icon = 'icons/obj/Cryogenic2.dmi'
 	icon_state = "sleeper_1"
 	outfit = /decl/hierarchy/outfit/lss_ert
-	ghost_role_perks = list(PERK_SI_SCI, PERK_CHEMIST, PERK_SURE_STEP, PERK_TRUE_NAME)
-	short_desc = "You are an Auditor."
-	flavour_text = "The upper colony has done something thats not upto code, whether it be LSS, SI or any other department your job is to look into any matter that HC tells you about. \
-	You're not a cop, you're not medical personal you're here to correct some papers not criminals."
-	assignedrole = "Guild Emergency Personnel"
-	title = "Guild Emergency Personnel"
+	ghost_role_perks = list(PERK_NO_OBFUSCATION, PERK_MARKET_PROF, PERK_TRUE_NAME)
```

Comment:
Why not chemist?

Reply:
Because they're specifically for Lonestar now, and don't really need it?